hack/prepare_agent_config.sh

```diff
@@ -9,9 +9,13 @@ echo "Found ethernet interface IP: ${IP}"
 
 echo "Creating agent config"
 cat <<EOF > bin/agent/etc/flightctl/config.yaml
-management-endpoint: https://${IP}:3333
-enrollment-endpoint: https://${IP}:3333
-enrollment-ui-endpoint: https://${IP}:8080
+enrollment-service:
+  authentication:
+    client-certificate: /etc/flightctl/certs/client-enrollment.crt
+    client-key: /etc/flightctl/certs/client-enrollment.key
+  service:
+    certificate-authority: /etc/flightctl/certs/ca.crt
+    server: https://${IP}:3333
```

Comment:
This is only convenience for our make deploy. We always deploy on 3333, the only thing that can change is the public IP of you devel machine or VM, which is what why we need a variable.

We cannot use 127.0.0.1 because the agent VM's will be their own localhost and couldn't reach the server.